src/api/app/presenters/notification_presenter.rb

```diff
@@ -31,4 +31,18 @@ def notification_badge
       'Comment'
     end
   end
+
+  def excerpt
```

Comment:
You can move each one of the `when`'s into the `@model` as a `excerpt` method and here just forward the `excerpt` to the `@model`

Reply:
@model is always a `Notification`'s object. Do you mean moving the `excerpt` method to that model? In that case, it will be identical, with the same `case` inside.

Or do you mean moving it to each notifiable model (Comment, Review, Request) ?
In such case, I don't know if it is an advantage since it's only used for notifications :thinking:

Reply:
I prefer if it stays here in the presenter since it's logic only for views dealing with notifications.

Reply:
> `@model` is always a `Notification`'s object. Do you mean moving the `excerpt` method to that model? In that case, it will be identical, with the same `case` inside.
> 
> Or do you mean moving it to each notifiable model (Comment, Review, Request) ?
> In such case, I don't know if it is an advantage since it's only used for notifications thinking

I meant on the notifiable object. But It's just a :thought_balloon:

Reply:
In fact, if we move `excerpt` to Notification model, applying the same logic, we'd have to do the same with the rest of the methods in the NotificationPresenter.

Reply:
Ok, so I think I can leave it in the NotificationPresenter. But I've simplified the code a bit, using `notifiable_type`instead of `event_type`. In case you want to have another look, @danidoni

Reply:
Okay!